master/buildbot/test/unit/test_www_hooks_bitbucket.py

```diff
@@ -150,12 +150,12 @@ def setUp(self):
 
     @inlineCallbacks
     def testGitWithChange(self):
-        change_dict = {'payload': [gitJsonPayload]}
+        change_dict = {b'payload': [gitJsonPayload]}
 
         request = FakeRequest(change_dict)
```

Comment:
I think we should also change FakeRequest.test_render to ensure that all its attributes are never using native strings. (method, args, uri, prepath, received_headers)

As per twisted spec, all Request args shall be bytes if I understand correctly.

That way we make sure we don't forget another place for this kind of bug.